packages/services/server/src/graphql-handler.ts

```diff
@@ -210,3 +234,42 @@ export const graphqlHandler = (options: GraphQLHandlerOptions): RouteHandlerMeth
     );
   };
 };
+
+if (!process.env['SUPERTOKENS_CONNECTION_URI']) {
+  throw new Error('Missing SUPERTOKENS_CONNECTION_URI env variable.');
+}
+
+if (!process.env['SUPERTOKENS_API_KEY']) {
+  throw new Error('Missing SUPERTOKENS_API_KEY env variable.');
+}
+
+const connectionUri = process.env['SUPERTOKENS_CONNECTION_URI'];
+const apiKey = process.env['SUPERTOKENS_API_KEY'];
```

Comment:
This decoding of the env variables should definetly be handled in a more elgant way...

Reply:
done!